selfdrive/car/toyota/carcontroller.py

```diff
@@ -29,7 +29,7 @@ def update(self, enabled, active, CS, frame, actuators, pcm_cancel_cmd, hud_aler
     if CS.CP.enableGasInterceptor and enabled:
       MAX_INTERCEPTOR_GAS = 0.5
       # RAV4 has very sensitive gas pedal
-      if CS.CP.carFingerprint in [CAR.RAV4, CAR.RAV4H, CAR.HIGHLANDER, CAR.HIGHLANDERH]:
+      if CS.CP.carFingerprint in [CAR.RAV4]:
```

Comment:
why was this changed?

Reply:
All these cars except the RAV4 are marked as `NO_STOP_TIMER_CAR`, so standstill isn't requested when the car stops, thus a pedal isn't needed to get them moving again if my understanding is correct.

Reply:
```
# no resume button press required
NO_STOP_TIMER_CAR = TSS2_CAR | set([CAR.RAV4H, CAR.HIGHLANDERH, CAR.HIGHLANDER, CAR.SIENNA, CAR.LEXUS_ESH])
```

Reply:
reverted this per #23991